installer/pkg/config-generator/generator.go

```diff
@@ -95,6 +101,105 @@ func (c *ConfigGenerator) TectonicSystem() (string, error) {
 	})
 }
 
+// InstallConfig returns, if successful, a yaml string for on-disk install-config from 4.0 installer.
```

Comment:
I'd reword to be more generic (no need to mention disks or 4.0).  Maybe something like:

> InstallConfig returns a YAML-rendered Kubernetes object with the user-supplied cluster configuration.

Reply:
done